addons/point_of_sale/static/tests/pos/tours/optional_product_tour.js

```diff
@@ -0,0 +1,40 @@
+import * as ProductScreen from "@point_of_sale/../tests/pos/tours/utils/product_screen_util";
+import * as Dialog from "@point_of_sale/../tests/generic_helpers/dialog_util";
+import * as Chrome from "@point_of_sale/../tests/pos/tours/utils/chrome_util";
+import * as OptionalProduct from "@point_of_sale/../tests/pos/tours/utils/optional_product_util";
+import { registry } from "@web/core/registry";
+
+registry.category("web_tour.tours").add("OptionalProductTour", {
```

Comment:
```suggestion
registry.category("web_tour.tours").add("test_05_optional_product", {
```

rename the tour with the name of the test.